stream_alert_cli/helpers.py

```diff
@@ -67,15 +68,81 @@ def run_command(runner_args, **kwargs):
     return True
 
 
-def continue_prompt(prompt=''):
-    """Continue prompt used to check user's response"""
+def continue_prompt(**kwargs):
```

Comment:
This helper function was modified to be more generic and is now used in more places than just the terraform generate code. This is a breaking change that impacts other areas; please change this function back to closer to what it is in master. The calling code should now perform necessary actions dependent upon the return value of this function. (ie:, `sys.exit(0)` if the return is `False`). Please see this code for additional context: https://github.com/airbnb/streamalert/blob/master/stream_alert_cli/config.py#L206

Function in master:
```
def continue_prompt(prompt=''):
    """Continue prompt used to check user's response"""
    required_responses = {'yes', 'no'}
    response = ''
    while response not in required_responses:
        prompt = prompt or 'Would you like to continue?'
        response = raw_input('\n{} (yes or no): '.format(prompt))

    return response == 'yes'
```
Please also make sure the callers check the return value and perform `sys.exit(0)` where necessary.